api.yaml

```diff
@@ -262,6 +263,8 @@ definitions:
     $ref: "https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/v0.39.0/api/definitions.yaml#/MessageResponseNotificationStatus"
   ProblemJson:
     $ref: "https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/v0.39.0/api/definitions.yaml#/ProblemJson"
+  Timestamp:
```

Comment:
@gunzip should we specify the format of this timestamp? (e.g. `IsoTimestamp`) ?

Reply:
no, it's implied by the swagger date-time specs:
https://swagger.io/docs/specification/data-models/data-types/#format